builds/misc/packages-release.yaml

```diff
@@ -984,13 +986,8 @@ stages:
                 keyVaultName: $(kv.name)
                 secretsFilter: >-
                   GitHubAccessToken
-            - bash: | #Create Release Page Before Publishing Artifacts in Parallel in the Next Job.
```

Comment:
Do we no longer need to create the Release Page here before the next job runs (by running the publishReleasePackages.sh script)? Was a change made to remove the need to do this? Or was it always unnecessary?

Reply:
The shell script that is being called can correctly handle the release page generation. It has a logic to detect if the release page is created or not. If there is no release page, it will create one along with uploading an artifact. (Plus, this run has always failed because by the time it gets to run these lines, the subsequential artifact upload has already started and created the release page.).

Reply:
Sure! It's cleaner that way.

Though I'm pretty sure that GitHub API won't do anything if we call the same url to create release page multiple times.